crawl-ref/source/util/server/install-ubuntu-chroot

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+#---help---
+# Usage: install-ubuntu-chroot -d destination-directory
+#
+# This script downloads and installs a recent version of Ubuntu to a chroot.
+#---help---
+
+set -euo pipefail
+
+die() {
```

Comment:
you could pull these commands out into a `common.sh` and source them for all scripts `source ./common.sh`

Reply:
I could, but I don't necessarily think it's worth it; these scripts are supposed to be relatively self contained